tonconnect-server/TonConnect2.md

```diff
@@ -0,0 +1,533 @@
+# TON Connect 2
+
```

Comment:
General feedback - this standard fails to address the most pressing issues we currently have in the ecosystem. People are trying to implement simple dapps that exist in other ecosystems like a Uniswap-like DEX. Before we go adding elaborate support for services, we should see that we have a decent support for this very simple use-case.

I don't see how the standard supports a simple DEX. A DEX is an app. It shouldn't have a backend. As a DEX developer, what do you suppose I do?

1. How can I connect a user and return the user to the browser that they came from on mobile (when this browser is possibly some native app)?

2. How am I supposed to handle the RPC endpoint? is it hardcoded in the app frontend? We agreed that this solution was horrible.